rootless/specconv/specconv_linux.go

```diff
@@ -67,6 +69,14 @@ func toRootless(spec *specs.Spec, v2Controllers []string, currentOOMScoreAdj int
 		}
 	}
 
+	// FIXME: Temporary workaround for https://github.com/docker/for-linux/issues/743.
+	//        Remove this once https://github.com/opencontainers/runc/issues/2091 is
+	//        resolved in runc.
+	if _, ok := spec.Linux.Sysctl["kernel.domainname"]; ok {
+		logrus.Info("Domainname is currently not supported when running in rootless mode. Discarding 'kernel.domainname' sysctl. See https://github.com/docker/for-linux/issues/743 for details.")
+		delete(spec.Linux.Sysctl, "kernel.domainname")
+	}
```

Comment:
Was in doubt what would be better; `discard` the option (and log, to make it slightly visible), or to return an informative error here?

@AkihiroSuda @cpuguy83 WDYT?